src/script/page/RightSidebar/ConversationDetails/utils/getConversationActions.ts

```diff
@@ -94,7 +94,10 @@ const getConversationActions = (
     {
       condition: isSingleUser && (userEntity?.isConnected() || userEntity?.isRequest()),
       item: {
-        click: () => actionsViewModel.blockUser(userEntity, true, nextConversationEntity),
+        click: () => {
+          actionsViewModel.blockUser(userEntity, true, nextConversationEntity);
+          closeRightPanel();
```

Comment:
I don't think the fix should be here. 
When we block a user there must be somewhere where we quit the conversation. 
Everytime we quit a conversation we should also close the right panel.